### ISSUE TITLE
Update Project Card for Access the Data

### ISSUE THREAD

**Issue by kathrynsilvaconway** [hackforla/website#3447]
### Overview
As a deleloper, I need the details of all Hack for LA projects to be up to date on the website. For this issue, we will update the Access the Data project card.

### Action Items

 - [x] Go to the file _projects/access-the-data.md
 - [x] Update the image to the  [current logo](https://drive.google.com/file/d/1GVGG8ie6eRbQUjUj2aM9tyPGhhUTs91W/view?usp=sharing)
  - [x] Replace the project description with the following:
  ```
         With government policies increasingly relying on data and data analysis, it becomes imperative to increase citizens' 
         access and understanding of local government data in order to be effective advocates for their communities.

         We are currently working on an MVP, which is a website that will serve as a foundation for the continued Access the Data 
         project. The MVP will consist of a website that hosts a catalog of sources of open government data. This project will 
         focus on Los Angeles County as its backdrop to test the hypothesis that this is a tool the community of community 
         activists and data experts needs and wants.
```
 - [x]  In the technologies section, add CKAN
 - [x] Remove the Partners section, as there is no current partner.
 - [x] In the tools section remove Sketch and phonecalls. Add [Google Drive](https://drive.google.com/drive/u/0/my-drive)
 - [x] In the links section, replace the Overview link with a link to the new [one sheet](https://drive.google.com/file/d/1uWSqc5jU6feY3XSi9g178qnoOkan6enH/view) (You 'll need to download the one sheet PDF first.)
 - [x] In the Resources section, replace the Overview link with a link to the new [one sheet](https://drive.google.com/file/d/1uWSqc5jU6feY3XSi9g178qnoOkan6enH/view)
 - [x] In the Current Project Team Section: 
     - [x] Remove: Sarah Nabelsi, Alyssa Bryant, Assel Abilova, Jeremy DeCarvalho, and Malak Hmimy
     - [x] Add:
     - name: AJ Price
    role: Product Manager
    links:
      slack: 'https://hackforla.slack.com/team/D02USE0EPSS
      github: 'https://github.com/mxajPrice'
    picture: https://avatars.githubusercontent.com/mxajPrice
     - name: Mariana Melo-Vega
    role: UX/UI Research Lead
    links:
      slack: 'https://hackforla.slack.com/team/U038BPUMFB7
      github: 'https://github.com/mmelovega'
    picture: https://avatars.githubusercontent.com/mmelovega
     - name: Kevin Chu
    role: UX/UI Design Lead
    links:
      slack: 'https://hackforla.slack.com/team/U037TKS7U9G
      github: 'https://github.com/KC243
    picture: https://avatars.githubusercontent.com/KC243
     - name: Judson Lester
    role: Developer Lead
    links:
      slack: 'https://hackforla.slack.com/team/U9NFW84QK
      github: 'https://github.com/nyarly
    picture: https://avatars.githubusercontent.com/nyarly
 - [x] Once every action item has been checked off, write to the stakeholder on issue #3153 asking them if they sign off on User Acceptance.

### Resources
- [PDF overview of Access the Data Project](https://drive.google.com/file/d/1uWSqc5jU6feY3XSi9g178qnoOkan6enH/view?usp=sharing) 
- [_projects/access-the-data.md](https://github.com/hackforla/website/blob/gh-pages/_projects/access-the-data.md)
- [Access the Data webpage](https://www.hackforla.org/projects/access-the-data)

### Associated Issue
 - #3153

**Comment by JessicaLucindaCheng** [comment]
@kathrynsilvaconway Is this from this issue https://github.com/hackforla/website/issues/3153? If yes, you don't need to re-write it. You can just check the original issue and see if it is "ready for milestone". This will be a little different than

**Comment by JessicaLucindaCheng** [comment]
@kathrynsilvaconway Is this from this issue https://github.com/hackforla/website/issues/3153? If yes, for these types of issues, I usually didn't rewrite them but just made sure that anything that **wasn't** checked off had information in a comment below as to what they wanted changed. If there was anything **not** checked off **and** they did **not** provide information for it in the comment below, I would @ mention the issue writer and ask them about it.

**Comment by JessicaLucindaCheng** [comment]
@kathrynsilvaconway Is this from this issue https://github.com/hackforla/website/issues/3153? If yes, for these types of issues, I usually didn't rewrite them but just made sure that anything in the original issue that **wasn't** checked off had information in a comment below as to what they wanted to be changed. If there was anything **not** checked off **and** they did **not** provide information for it in the comment below, I would @ mention the issue writer and ask them about it. If needed, I would edit the issue wording a little if something was really confusing. Once it was ready for it, I would add a "ready for milestone" label.

You don't have to do it the way I did it and if you want to re-write them you can. I'm not trying to micro-manage how you do things. Also, I'm sorry if my adding the `ready for dev lead` label confused you as to what to do with it.

Also, the format of that issue (https://github.com/hackforla/website/issues/3153) comes from the template that is for other teams to ask for updates on their info on the website: https://github.com/hackforla/website/issues/new?assignees=&labels=P-Feature%3A+Project+Info+and+Page%2C+role%3A+back+end%2FdevOps%2C+role%3A+front+end%2C+Size%3A+Small%2C+size%3A+0.5pt%2C+time+sensitive&template=project-profile-card-review-and-update.md&title=Update+Project+Profile%3A+%5BProject+Name%5D

**Comment by kathrynsilvaconway** [comment]
> @kathrynsilvaconway Is this from this issue #3153? If yes, for these types of issues, I usually didn't rewrite them but just made sure that anything in the original issue that **wasn't** checked off had information in a comment below as to what they wanted to be changed. If there was anything **not** checked off **and** they did **not** provide information for it in the comment below, I would @ mention the issue writer and ask them about it. If needed, I would edit the issue wording a little if something was really confusing. Once it was ready for it, I would add a "ready for milestone" label.
> 
> You don't have to do it the way I did it and if you want to re-write them you can. I'm not trying to micro-manage how you do things. Also, I'm sorry if my adding the `ready for dev lead` label confused you as to what to do with it.

Thanks for your input, it is always helpful. When I started trying to add the action items to #3153, I felt like I was just rewriting the entire issue. It seemed more effective to just make a new one.

This one feels more like two separate issues to me. #3153 is a request submitted for changes to be made, and the issue I wrote is a list of actionable items for devs. I don't want to over-simplify tasks, but it feels clearer to me. If @ExperimentsInHonesty wants it done differently, I can certainly change the approach. My thought was to link this issue to #3153 and just close that one with a comment.

**Comment by bishrfaisal** [comment]
@ExperimentsInHonesty Issue is ready for milestone and good to go. Please advise on merging this issue with #3153 or keeping it separate as it is.

**Comment by 7kram** [comment]
@ExperimentsInHonesty Really organized and clear! I don't think #3153 is an issue as long it refers to this one. I would suggest adding "closing issue #3153" as an Action Item once everything else has been completed. 

Note: There is a "time-sensitive" label but I'm not seeing a reference to when it should be completed.

Bonnie's A: 
- #3153 will go into the UAT column as soon as the changes have been made and are live on the site and we will reach out to the Product Manager of that project to confirm they are satisfied.
- Time Sensitive: Anything that is marked time sensitive should be done immediately.  It moves it out of the milestone priorities into immediate action required. If there is more than one, we work on them in order of their milestones.

**Comment by ExperimentsInHonesty** [comment]
@kathrynsilvaconway There is a new [one sheet](https://drive.google.com/file/d/1uWSqc5jU6feY3XSi9g178qnoOkan6enH/view) (also known as an overview page) listed in https://github.com/hackforla/website/issues/3153#issuecomment-1182507813  

It will have to be downloaded by the person working on this issue, and then put in the proper directory in our repo. Please revise the steps in the action items accordingly. 

Please add `ready for Milestone` label again when you are ready for another review

**Comment by kathrynsilvaconway** [comment]
@ExperimentsInHonesty I added the action item to close #3153 . Also, updating the overview link with the new one sheet.

**Comment by ExperimentsInHonesty** [comment]
- I have updated the issue to say to follow back up with the Stakeholder once the PR has been merged on issue #3153

**Comment by mariareeves** [comment]
i. Availability: Mon-Thus 12-13pm and Friday 9-13pm 
ii. ETA: 9/30/22

**Comment by mariareeves** [comment]
I have a blocker. I am having trouble pushing my branch - I am able to see the changes I made in the website using docker . Here is what I have done:

- I made a commit
- then I used the command git push --set-upstream origin update-project-card-3447
- then there were some conflicting changes and I tried git fetch upstream
git checkout -b upstream-gh-pages --track upstream/gh-pages as the "CONTRIBUTING" file said. 
 - then when I try to push it again says "Everything up-to-date". It seems like there are  no changes, although I can see the changes I made in the website when I run docker. 
![tempImageo8TGpW](https://user-images.githubusercontent.com/109241790/192336826-d9126562-d9e2-4378-b56c-a683a437c6fe.gif)

![tempImagey3rYyV](https://user-images.githubusercontent.com/109241790/192337021-7c2537bd-2a3a-42ad-bb35-c2ec3ff66ee6.gif)

 
I am working in the 'update-project-card-3447' branch.

Any assistance is appreciated. Thank you.
